frontend/src/Components/Loader/Loader.jsx

```diff
@@ -1,16 +1,14 @@
 import React from "react";
 import "./Loader.css";
 
-const Loader = () => {
-	return (
-		<div className="container-fluid d-flex flex-column justify-content-center align-items-center vh-100">
-			<div className="spinner-border text-primary loader-circle-size" role="status">
-				<span className="sr-only">Loading...</span>
-			</div>
-			<br />
-			<div className="text-muted loader-text-size">Loading your data...</div>
+const Loader = () => (
+	<div className="container-fluid d-flex flex-column justify-content-center align-items-center vh-100">
+		<div className="spinner-border text-primary loader-circle-size" role="status">
+			<span className="sr-only">Loading...</span>
 		</div>
-	);
-};
+		<br />
+		<div className="text-muted loader-text-size">Loading your data...</div>
```

Comment:
Why having both Loading... and Loading your data...?
Also, break on new line can be done with CSS, but it's no big deal.

Reply:
Fixed, thanks.